src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -305,9 +300,8 @@
     //==================================================================================================================
```

Comment:
While you're at it, you should really extract a `FuncotatorArgumentCollection` containing all the tool arguments.

Reply:
Fixed!